avocado_vt/loader.py

```diff
@@ -16,6 +16,7 @@
 Avocado VT plugin
 """
 
```

Comment:
@ldoktor I take what I wrote above back.  I had the impression that `--vt-arch` was being added to `list` outside of `add_basic_vt_options()`, and thus, it was an additional change.  Sorry for the confusion.